config/development.js

```diff
@@ -10,6 +10,11 @@ module.exports = {
   enableFileLogs: false,
   enableConsoleLogs: true,
 
+  githubApi: {
```

Comment:
We dont need to add this to development and test config unless the values are changing.